### PR TITLE
AF-1830: Package Name White List should contain Projects default package

### PR DESCRIPTION
**Pull request by jesuino**
AF-1830: Package Name White List should contain Projects default package

### CODE REVIEW

File: kie-wb-common-services/kie-wb-common-services-backend/src/main/java/org/kie/workbench/common/services/backend/project/ModuleSaver.java [hunk 1]
```diff
@@ -150,7 +150,8 @@ private void createModule() {
             projectImportsService.saveProjectImports(simpleModuleInstance.getImportsPath());
 
             //Create Module configuration - project package names White List
-            packageNameWhiteListService.createModuleWhiteList(simpleModuleInstance.getPackageNamesWhiteListPath());
+            packageNameWhiteListService.createModuleWhiteList(simpleModuleInstance.getPackageNamesWhiteListPath(),
```
Comment (ederign) [inline kie-wb-common-services/kie-wb-common-services-backend/src/main/java/org/kie/workbench/common/services/backend/project/ModuleSaver.java]: formating.

File: kie-wb-common-services/kie-wb-common-services-backend/src/main/java/org/kie/workbench/common/services/backend/project/ModuleSaver.java [hunk 2]
```diff
@@ -196,5 +197,9 @@ private void createDefaultPackage() {
         private Package getDefaultPackage() {
             return resourceResolver.resolvePackage(Paths.convert(Paths.convert(moduleRoot).resolve(MAIN_RESOURCES_PATH)));
         }
+        
+        public String defaultPackageNamesWhiteListEntry() {
```
Comment (ederign) [inline kie-wb-common-services/kie-wb-common-services-backend/src/main/java/org/kie/workbench/common/services/backend/project/ModuleSaver.java]: could you please add tests for this?

Comment (jesuino) [reply kie-wb-common-services/kie-wb-common-services-backend/src/main/java/org/kie/workbench/common/services/backend/project/ModuleSaver.java]: Could the test below cover this method? The method createModuleWhiteList is called with the response from defaultPackageNamesWhiteListEntry and we check if the correct value is passed to it.

Thanks!

**Comment by barboras7** [comment]
Jenkins execute full downstream build

**Comment by barboras7** [comment]
@ederign Could we please merge this?

**Comment by jesuino** [comment]
Added a new test in commit e15f9b084334b3e4b2074ecabeac68bb0c7e9607

**Comment by jesuino** [comment]
Jenkins execute full downstream build

**Pull request merged**